### DISCUSSION TITLE
How to receive media files through incoming webhook

### DISCUSSION THREAD

**Discussion by mhdhhemant** [LiveHelperChat/livehelperchat#2089] [Q&A]
I want to receive media files like images, videos, pdf through the incoming webhook. I went through the incoming webhook configuration as provided on https://doc.livehelperchat.com/docs/integrating/waha but was not able to receive it. Please help.

**Comment by remdex** [comment]
So just try to debug and see where you fail. You can enable incoming webhook loggin in audit log etc... There is many intergrations now where you can look just how they are done...

**Comment by mhdhhemant** [comment]
I tried to enable logs for incoming webhooks but was not able to. however. Please suggest how to.

**Comment by remdex** [comment]
I have nothing to sugest the is up to you... If you can't replicate samples, just hire someone :)
